Disable image description in lightbox

ibrkhalil opened the pull request:
fixes https://github.com/status-im/status-mobile/issues/16401#issuecomment-1609041213
status: ready

ibrkhalil committed:
Disable image description in lightbox

status-im-auto commented:
### Jenkins Builds
| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | f8dd7351 | [#1](https://ci.status.im/job/status-mobile/job/prs/job/android-e2e/job/PR-16405/1/) | 2023-06-27 08:49:36 | ~6 min | `android-e2e` | [:robot:`apk`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-230627-084253-f8dd73-pr16405-x86.apk) [:calling:](https://chart.apis.google.com/chart?cht=qr&chs=400x400&chld=L%7C%0A1&chl=https%3A%2F%2Fstatus-im-mobile-prs.ams3.cdn.digitaloceanspaces.com%2FStatusIm-Mobile-230627-084253-f8dd73-pr16405-x86.apk)|

OmarBasem reviewed (approved):
Wasn't this already done in a previous PR I think?

ibrkhalil commented:
> Wasn't this already done in a previous PR I think?

Yes I fixed a bug related to rendering mentions and text messages in the description and added it again by accident.
